components/admin/lessons/AdminLessonInfo.tsx

```diff
@@ -152,14 +153,15 @@ export const AdminLessonInfo: React.FC<LessonInfoProps> = ({
   selectedLesson
 }) => {
   // true when user clicks on `create new lesson` button
-  if (lessons && selectedLesson === lessons.length) {
+  if (!lessons || selectedLesson === lessons.length) {
```

Comment:
Without check for lessons this component won't handle empty Lesson data. This is unlikely to happen in production but since lesson type is `Lesson[]|undefined` I need to cover for it.

Reply:
Put this in the comment for the code and not in the PR.